Gemfile

```
@@ -14,3 +14,4 @@ gem 'pygments.rb', '2.2.0'
 gem 'thread_safe', '0.3.6'
 gem 'epubcheck-ruby', '4.2.5.0'
 gem 'html-proofer', '3.19.2'
+gem 'kindlegen'
```

```suggestion
gem 'kindlegen', '3.1.1'
```

How about we pin to the current latest release of `kindlegen`? This way everybody uses the same version of `kindlegen` and it ensures our builds are reproducible later on, in case we need to debug between versions. 😉 
Dependabot will also raise update PRs for the `kindlegen` package, so we can update with minimal fuss.

---

Sure, fixed.